Added Admin routing if admin logs in.

karynnglli opened the pull request:
NO ROUTE PROTECTION ENFORCED YET!
If a user logs in and the isAdmin boolean = false, then the user is shown the "dashboard"
If a user logs in and the isAdmin boolean = true, then the user is shown the "adminDashboard"

karynnglli committed:
Added Admin routing if admin logs in.
NO ROUTE PROTECTION ENFORCED YET!

YeshaP reviewed (commented):
should we put in /admin like the actual route instead of the dummy you have going for now and maybe just drag it all over into the public folder? idk if we can do that atm or not lol

karynnglli committed:
Accepted YP's comment and confirmed that it works

Commented out: res.redirect('/adminDashboard');
Added line: res.redirect('/admin.html');

YeshaP reviewed (commented):
I'm confused why is it admin.html lol isnt it supposed to be admin.tsx?

karynnglli commented:
> should we put in /admin like the actual route instead of the dummy you have going for now and maybe just drag it all over into the public folder? idk if we can do that atm or not lol

Hi YP, good idea and yes we can. I have modified the code :-)

karynnglli commented:
> I'm confused why is it admin.html lol isnt it supposed to be admin.tsx?

Hi, yes, it should be, but I needed to test that the change would work but I do not know tsx, so I tested with an HTML file and it confirmed to me that the routing works. we can change it at anytime, but just so that we do not get any errors and forget. It is just a placeholder for now. :-)
